lib/jobba/status.rb

```diff
@@ -6,6 +6,14 @@ class Status
 
     include Jobba::Common
 
+    def self.all
+      Query.new
+    end
+
+    def self.where(*args)
+      all.where(*args)
+    end
```

Comment:
I could have both `Jobba` and `Jobba::Status` delegate to `Jobba::Query` if that's preferable...